src/sdk.js

```diff
@@ -1,42 +1,42 @@
-import axios from 'axios';
-import _ from 'lodash';
-import { fnPath as urlPathToFnPath, trimEndSlash, formData } from './utils';
-import paramsSerializer from './params_serializer';
-import AddAuthHeader from './interceptors/add_auth_header';
-import RetryWithRefreshToken from './interceptors/retry_with_refresh_token';
-import RetryWithAnonToken from './interceptors/retry_with_anon_token';
-import ClearTokenAfterRevoke from './interceptors/clear_token_after_revoke';
-import FetchRefreshTokenForRevoke from './interceptors/fetch_refresh_token_for_revoke';
-import AddAuthTokenResponse from './interceptors/add_auth_token_response';
-import SaveToken from './interceptors/save_token';
-import FetchAuthTokenFromApi from './interceptors/fetch_auth_token_from_api';
-import FetchAuthTokenFromStore from './interceptors/fetch_auth_token_from_store';
-import AddClientIdToParams from './interceptors/add_client_id_to_params';
-import AddClientSecretToParams from './interceptors/add_client_secret_to_params';
-import AddIdpClientIdToParams from './interceptors/add_idp_client_id_to_params';
-import AddIdpIdToParams from './interceptors/add_idp_id_to_params';
-import AddIdpTokenToParams from './interceptors/add_idp_token_to_params';
-import AddSubjectTokenToParams from './interceptors/add_subject_token_to_params';
-import AddGrantTypeToParams from './interceptors/add_grant_type_to_params';
-import AddTokenExchangeGrantTypeToParams from './interceptors/add_token_exchange_grant_type_to_params';
-import AddScopeToParams from './interceptors/add_scope_to_params';
-import AuthInfo from './interceptors/auth_info';
-import MultipartRequest from './interceptors/multipart_request';
-import TransitRequest from './interceptors/transit_request';
-import TransitResponse from './interceptors/transit_response';
-import { createDefaultTokenStore } from './token_store';
-import contextRunner from './context_runner';
+import axios from "axios";
```

Comment:
@rap1ds @OtterleyW 

I guess my VS code has very different idea of how we should format JS. I see `.eslintrc.js` on the root. Is that enough? I guess I need to somehow make my VS follow that lint config.

Reply:
yarn format can fix styling issue.